roles/openshift_logging_elasticsearch/vars/main.yml

```diff
@@ -5,6 +5,8 @@ __kibana_index_modes: ["unique", "shared_ops"]
 
 __es_local_curl: "curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key"
 
+__elasticsearch_ready_retries: "{{ openshift_logging_elasticsearch_poll_timeout_minutes | default(20) | int * 2 }}"
```

Comment:
So we multiple by 2 because we have a 30 second delay?

Reply:
Correct